From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Luminoso-256 <63971285+Luminoso-256@users.noreply.github.com>
Date: Sat, 27 Feb 2021 12:19:17 -0600
Subject: [PATCH] Swap sunset modal for legal notice modal


diff --git a/assets/resource_packs/genoa_base/texts/en_US.lang b/assets/resource_packs/genoa_base/texts/en_US.lang
index 348cdaea906518e1a89c7357b15b7a7bdb6b5f28..01691539cc51c0819dd7af9169fa9aeaa4d290a7 100644
--- a/assets/resource_packs/genoa_base/texts/en_US.lang
+++ b/assets/resource_packs/genoa_base/texts/en_US.lang
@@ -6310,11 +6310,11 @@ map.tappable.limit.content=You've collected too many tappables too quickly. You
 map.xp.increment=+%s XP
 map.ugcpermission.content=You cannot perform this action because of how your Xbox Live account is setup. This can be changed in your privacy & online safety settings on Xbox.com.
 
-sunset.overlay.header=Time to Sunset
+sunset.overlay.header=Legal Notice!
 sunset.overlay.button=Sunset Information
 sunset.overlay.button.redeem=Redeem Free Copy of Minecraft
 sunset.modal.title=Time to Sunset
-sunset.modal.body=Thanks for playing Minecraft Earth! We have made the difficult discussion to end support for Minecraft Earth on June 30th 2021. Thanks for playing and we hope to see you in Minecraft!
+sunset.modal.body= Project Earth is NOT an official Minecraft product. Project Earth is not affiliated with, Endorsed by, or Associated with Mojang Studios or Microsoft. Minecraft and Minecraft Earth are Trademarks of Mojang Studios.
 sunset.modal.button.one=Sunset Information
 sunset.modal.button.two=Good to Know
 sunset.settings.button=Sunset Information
